content/general/authentication.textile

```diff
@@ -207,6 +170,24 @@ ably.auth.requestToken({ clientId: 'client@example.com' }, function(err, tokenDe
 
 *Note:* The authenticity of the JWT *will not* be checked, due to Ably not having access to your SECRET key.
 
+h2(#selecting-auth). Selecting an authentication mechanism
```

Comment:
All of this content below is missing links, for example Token Authentication could link to Tokemn Authentciation, identified links to understanding identified clients.  Please remember to hyperlink so that users can navigate our docs contextually easily.